### DISCUSSION TITLE
[@types/react-native]: TextInput prop `autoComplete` is outdated

### DISCUSSION THREAD

**Discussion by chwallen** [DefinitelyTyped/DefinitelyTyped#64985] [Issues with a @types package] [answered]
Starting with React Native 0.71, the `autoComplete` prop is now available for iOS. However, the types library is on 0.71.5 and has still not incorporated this change. [Here](https://reactnative.dev/docs/textinput#autocomplete) is a link to the available options. Some are available for both iOS and Android while others are platform-exclusive.

[This](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/d97582c92665bd5e829e6f1df5e85f3c417b4606/types/react-native/Libraries/Components/TextInput/TextInput.d.ts#L318) is where the `autoComplete` prop is currently defined.

[a comment by an automated account is not shown]

**Comment by lunaleaps** [comment]
Thanks for reporting, do you want to submit a PR for this?

**Reply by chwallen** [reply, marked as answer]
I submitted a PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65144.
